ui/components/notification.js

```diff
@@ -0,0 +1,50 @@
+import { Fragment } from 'react'
+import { Transition } from '@headlessui/react'
+import { CheckCircleIcon } from '@heroicons/react/outline'
+import { XIcon } from '@heroicons/react/solid'
+
+export default function Notification ({ show, setShow, text }) {
+  return (
+    <>
+      <div
+        aria-live='assertive'
+        className='fixed inset-0 flex items-end px-4 py-6 pointer-events-none sm:p-6 sm:items-start'
+      >
+        <div className='w-full flex flex-col items-center space-y-4 sm:items-end fixed bottom-[5%] right-[1%]'>
```

Comment:
Would replace `5%` and `1%` with fixed values - since it's hard to predict how it will behave on larger or smaller screens